2022/cloudexperts/index.md

```diff
@@ -0,0 +1,10 @@
+# Cloud Experts
+
+Visita: https://www.cloudexperts.pe
```

Comment:
The link https://[www.cloudexperts.pe](https://www.cloudexperts.pe/)/ does not seem to lead anywhere.

Reply:
@frankchambillo !

Reply:
please remove that line

Reply:
I corrected the file but I don't know if it's ok